Core/include/Acts/EventData/MultiComponentTrackParameters.hpp

```diff
@@ -67,11 +66,55 @@ class MultiComponentBoundTrackParameters {
   }
 
  public:
+  using ConstructionTuple = std::tuple<double, Acts::Vector4, Acts::Vector3,
```

Comment:
Struct with named members? Can still use structured bindings.

Reply:
I moved this up from `MultiComponentCurvilinearTrackParameters`. I believe the multi stepper infrastructure has quite some `using` here and there. In that sense it is consistent right now. Potentially something we want to follow up on in general.

Reply:
ok